tutorials/spanruler_restaurant_reviews/scripts/rules.py

```diff
@@ -0,0 +1,495 @@
+from typing import Dict, List, Any
+from spacy.util import registry
+
+
+Rules = List[Dict[str, Any]]
+
+
+@registry.misc("restaurant_rules.v1")
+def restaurant_span_rules() -> Rules:
```

Comment:
The way this is set up, perfectly allows for a `.v2` in which we may keep some of the rules and maybe others not, if they don't improve overal performance.